README.md

```diff
@@ -5,6 +5,32 @@
 [![Validation](https://github.com/p4lang/p4c/actions/workflows/ci-validation-nightly.yml/badge.svg)](https://github.com/p4lang/p4c/actions/workflows/ci-validation-nightly.yml)
 [![Docker Container](https://github.com/p4lang/p4c/actions/workflows/ci-container-image.yml/badge.svg)](https://github.com/p4lang/p4c/actions/workflows/ci-container-image.yml)
 
+
+p4c
+=================
+
+* [Getting started](#getting-started)
+   * [Installing packaged versions of p4c](#installing-packaged-versions-of-p4c)
+   * [Installing p4c from source](#installing-p4c-from-source)
+* [Dependencies](#dependencies)
+   * [Ubuntu dependencies](#ubuntu-dependencies)
+   * [Fedora dependencies](#fedora-dependencies)
+   * [macOS dependencies](#macos-dependencies)
+   * [Garbage collector](#garbage-collector)
+   * [Crash dumps](#crash-dumps)
+* [Development tools](#development-tools)
+   * [Git setup](#git-setup)
+* [Docker](#docker)
+* [Bazel](#bazel)
+* [Build system](#build-system)
+   * [Defining new CMake targets](#defining-new-cmake-targets)
+* [Known issues](#known-issues)
+   * [Frontend](#frontend)
+   * [Backends](#backends)
+* [How to Contribute](#how-to-contribute)
+* [P4 Compiler Onboarding](#p4-compiler-onboarding)
+* [Contact](#contact)
+
 # p4c
```

Comment:
Remove this then we are done. We want p4c at the top and the second header is superfluous. 

![image](https://github.com/p4lang/p4c/assets/5960321/87cab030-187e-4729-a1f1-62949ced32df)

Reply:
Sure Sir

> 😅Just finished writing the same question on above review. That what to do with this heading.